### ISSUE TITLE
Enabling the magnifier to see log errors when connect/read is enabled

### ISSUE THREAD

**Issue by fabiofilz** [tchiotludo/akhq#614]
Hi,

I enabled the connect/read role, but the magnifying glass icon in the connector screen is not showing so the user with read-only access cannot see the connector's parameters, tasks or the error logs.

Basically the same as connect/update, but not enabling the value fields and update button.

Would it be possible?

Thank you.
Fabio

**Comment by tchiotludo** [comment]
Seems to be possible :) 
Can you share screnshot to be sure to understand ?

**Comment by fabiofilz** [comment]
Hi @tchiotludo,

Yes, sure!

Using connect/read role

![image](https://user-images.githubusercontent.com/45266582/109859002-63e54900-7c54-11eb-9262-6a86897f5070.png)


Using connect/update role
![image](https://user-images.githubusercontent.com/45266582/109856187-1d421f80-7c51-11eb-9584-de36bd1bcb65.png)

Tasks
![image](https://user-images.githubusercontent.com/45266582/109856347-4c589100-7c51-11eb-944c-ef69067431a8.png)

This is the Update button
![image](https://user-images.githubusercontent.com/45266582/109856448-685c3280-7c51-11eb-83d1-4ae23a8d012a.png)

Thank you.
Fabio

**Comment by tchiotludo** [comment]
Ok will be simple :) Thanks :)

**Comment by tchiotludo** [comment]
You can try on dev version 
fix by #620

**Comment by fabiofilz** [comment]
Hi @tchiotludo,

Could you please create a new TAG with the changes, please?

I checked the Dockerhub, but I didn't see a new TAG. The 0.0.17 was created a month ago.

Thank you.
Fabio

![image](https://user-images.githubusercontent.com/45266582/111539021-51dfcc00-8765-11eb-8666-1ab3a4d59571.png)

**Comment by tchiotludo** [comment]
this one :smile: : 
![image](https://user-images.githubusercontent.com/2064609/111539214-e5b59600-876d-11eb-862a-2b763b4a888e.png)
the `dev` version ;)

**Comment by fabiofilz** [comment]
Hi @tchiotludo,

Sorry, I thought it was a final release with the changes.

I have just deployed the DEV tag and I am using the role connect/read.

I am still not able to see the magnifier to access the error logs and I also notice that when I change between one Kafka Connect and the other it is not refreshing the page (list of connectors). It was not happening in the previous version.

![image](https://user-images.githubusercontent.com/45266582/111540662-56a57f80-8767-11eb-9ccc-4fe11cd48618.png)

![image](https://user-images.githubusercontent.com/45266582/111541020-d9c6d580-8767-11eb-851f-b54726b485cd.png)

Thank you.
Fabio

**Comment by tchiotludo** [comment]
@PLarboulette does this may due to recent change ?

**Comment by tchiotludo** [comment]
or @alozano3 ? :smile:

**Comment by alozano3** [comment]
Hi, I deployed the latest dev version and the magnifier is working with one connect cluster.
I will try to have two clusters to try to reproduce the issue, but I don't think it is related with #620

**Comment by fabiofilz** [comment]
Hi @alozano3 , @tchiotludo ,

I tested the DEV version and the magnifier is working fine, the fields are disabled and the buttons (update, delete, restart and pause) are hidden when using connect/read. I also tested with the connect/update, connect/insert, connect/delete, connect/state/update and they were working fine.

The only issue that I can see is when changing between KC instances, it updates the URL and the title at the top Connect : <KC instance name>, but the page is not refreshed. (If a press the browser button to refresh it works).

Thank you.
Fabio
